inc/invitation.class.php

```diff
@@ -114,6 +114,13 @@ public function prepareInputForAdd($input) {
          return false;
       }
 
+      // does the user have a default email?
+      if (!$user->getDefaultEmail()) {
```

Comment:
the list of users for adding an invitation shows all users without checking if their email is assigned

Reply:
hi,

don't fix this here : there is a workflow change in progress.

Reply:
I checked: the workflow change was merged. We might do better by filtering users in the dropdown when we create an invitation, but for now, I accept this.